package.json

```diff
@@ -23,8 +23,11 @@
   },
   "homepage": "https://github.com/founders-programme-2/business-launchpad#readme",
   "dependencies": {
+    "bcrypt": "^3.0.6",
     "concurrently": "^4.1.0",
```

Comment:
What this? why do we want to use it?

Reply:
it's for hashing the password. what did you use in your projects ?

Reply:
exactly! for hashing passwords before they are stored in the database.
Here are the docs:
https://www.npmjs.com/package/bcrypt